src/envoy/auth/jwt_test.cc

```diff
@@ -363,6 +364,18 @@ class JwtTest : public testing::Test {
       EXPECT_TRUE(EqJson(payload, jwt.Payload()));
     }
   }
+
+  //Test whether the sub field in the JWT is as expected
+  void DoJwtSubEqualTest(std::string jwt_str, std::string sub_expected,
```

Comment:
It is better not to write this function, instead just write these 2 or 3 times in each test.  It is easier to read and if anything fails, it is easy to debug